pulp_container/app/models.py

```diff
@@ -200,6 +200,50 @@ class Meta:
         unique_together = (("name", "tagged_manifest"),)
 
 
+class ImageManifestSignature(Content):
+    """
+    A signature for an image manifest.
+
+    Fields:
+        name (models.CharField): A signature name in the 'manifest_digest@random_name' format
+        digest (models.CharField): A signature sha256 digest.
+        signature_type (models.CharField): A signature type as specified in signature metadata.
+                                           Currently it's only "atomic container signature".
+        signing_key_id (models.CharField): A key id identified by gpg (last 8 bytes of the
+                                           fingerprint).
+        signature_timestamp (models.PositiveIntegerField): A signature timestamp identified by gpg.
+
+
+    Relations:
+        signed_manifest (models.ForeignKey): An image manifest this signature is relevant to.
+
+    """
+
+    PROTECTED_FROM_RECLAIM = False
+
+    TYPE = "signature"
+
+    SIGNATURE_CHOICES = ((SIGNATURE_TYPE.ATOMIC_SHORT, SIGNATURE_TYPE.ATOMIC_SHORT),)
+
+    name = models.CharField(max_length=255)
+    digest = models.CharField(max_length=255)
+    signature_type = models.CharField(max_length=255, choices=SIGNATURE_CHOICES)
```

Comment:
this is probably matter of preference, i'd not prepend those fields wit `signature_` since they are already on the signature model